src/templates/next/layouts/Content/Content.stories.tsx

```diff
@@ -32,6 +32,10 @@ Default.args = {
       siteNavItems: [],
     },
     lastUpdated: "1 Jan 2021",
+    search: {
```

Comment:
confused why these are localSearch for Collection and Content layout instead of searchSG?
I thought it's a site wide thing that is used for the navbar on the entire site

Reply:
This is inside the context of a story so I kept it to local search so I could avoid duplicating the `useEffect` logic + have different layouts to visualise how the local search and SearchSG looks like within a page.